drivers/hmis/app/graphql/mutations/create_beds.rb

```diff
@@ -5,9 +5,10 @@ class CreateBeds < BaseMutation
     field :inventory, Types::HmisSchema::Inventory, null: true
 
     def resolve(input:)
-      inventory = Hmis::Hud::Inventory.editable_by(current_user).find_by(id: input.inventory_id)
+      inventory = Hmis::Hud::Inventory.viewable_by(current_user).find_by(id: input.inventory_id)
       errors = HmisErrors::Errors.new
       errors.add :inventory_id, :not_found unless inventory.present?
+      return { inventory: nil, errors: [HmisErrors::Error.new(:inventory_id, :not_allowed)] } unless current_user.permissions_for?(inventory, :can_edit_project_details)
```

Comment:
FYi, in all these places the `<record name>: nil` is not needed. GraphQL will fill it in with a null if it's not returned. 
I prefer excluding to decrease the line width. I didn't realize that until recently, so that's why there is a mix of both approaches throughout the mutations.

Just a suggestion, not necessary to update unless you want to.